example/src/main/java/org/secondbase/example/main/HelloSecondBase.java

```diff
@@ -15,8 +15,11 @@ private void printVariable() {
     }
 
     public static void main(final String[] args) throws SecondBaseException {
+        final String[] realArgs = new String[] {
+                "--service", "test"
+        };
         final HelloSecondBase helloSecondBase = new HelloSecondBase();
-        new SecondBase(args, new Flags().loadOpts(helloSecondBase));
+        new SecondBase(realArgs, new Flags().loadOpts(helloSecondBase));
```

Comment:
"service" is a mandatory flag? I've missed that in reviews. Then we're back to the whole --coordinate debacle. The most used flags in Base was --disable-cloudname and --disable-logsetup due to them being forced upon the user. Could we change the behaviour of logging to only use service when it's been set?